### PR TITLE
#21 Set package metadata ie repo url

### LINKED ISSUE #21: [FEATURE REQUEST]: Set Package metadata
### Is your feature request related to a problem? Please describe.

I want to have a rich  as possible developer experience which means providing additional meta data as part of nuget package

### Describe the solution you'd like

The licence expression and repo url should be set

### Describe alternatives you've considered

Accept info is missing

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by thompson-tomo**
Sets both the repo url & the license expression

Closes #21

### CODE REVIEW

**Comment by nojaf** [comment]
Hi @adamreeve, could you take a look at this?

File: ParquetSharp.DataFrame/ParquetSharp.DataFrame.csproj [hunk 1]
```diff
@@ -16,16 +16,11 @@
     <Copyright>Copyright G-Research 2021</Copyright>
     <PackageProjectUrl>https://github.com/G-Research/ParquetSharp.DataFrame</PackageProjectUrl>
     <PackageTags>dataframe apache parquet gresearch g-research .net c#</PackageTags>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/G-Research/ParquetSharp.DataFrame</RepositoryUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
   </PropertyGroup>
 
-  <ItemGroup Label="Additional nuget files">
```
Comment (adamreeve) [inline ParquetSharp.DataFrame/ParquetSharp.DataFrame.csproj:22]: Can you please keep the license file included in the NuGet package

Comment (thompson-tomo) [reply ParquetSharp.DataFrame/ParquetSharp.DataFrame.csproj:22]: done

**Review by adamreeve — APPROVED** [review]

**Comment by nojaf** [comment]
Thanks @thompson-tomo!

**Pull request merged**